pipeline/frontend/yaml/linter/schema/schema_test.go

```diff
@@ -101,6 +101,11 @@ func TestSchema(t *testing.T) {
 			testFile: ".woodpecker/test-broken.yml",
 			fail:     true,
 		},
+		{
+			name:     "New syntax",
+			testFile: ".woodpecker/test-new-syntax.yml",
+			fail:     false,
+		},
```

Comment:
I would suggest to call the file sth like `test-array-steps.yml` or `test-array-syntax.yml`.